protocols/407d5e/README.md

```diff
@@ -0,0 +1,82 @@
+# Protein  Labeling with Incubation
```

Comment:
Not urgent: I just noticed an extra space after 'Protein'. Does this affect the styling when it's pushed out or does it correct itself since it's in markdown?